XcodeServerSDK/XcodeServer.swift

```diff
@@ -415,6 +447,32 @@ public extension XcodeServer {
         }
     }
     
+    /**
+    XCS API call for retrievieng all available integrations on server.
+    
+    - parameter integrations:   Optional array of integrations.
+    - parameter error:          Optional error.
+    */
+    public func getIntegrations(completion: (integrations: [Integration]?, error: NSError?) -> ()) {
```

Comment:
WOW, this is pretty cool! I'll add it for sure but I guess after #47 is complete so I don't get lost 😆